conans/client/rest/rest_client.py

```diff
@@ -144,7 +144,7 @@ def get_package_info(self, package_reference):
         contents = {key: decode_text(value) for key, value in dict(contents).items()}
         return ConanInfo.loads(contents[CONANINFO])
 
-    def get_recipe(self, conan_reference, dest_folder):
+    def get_recipe(self, conan_reference, dest_folder, retrieve_sources=False):
```

Comment:
I really prefer a generic "skip_files=[]" and keep **rest_client** nescient about the EXPORT_SOURCES_TGZ_NAME